lib/command/delete.rb

```diff
@@ -36,7 +36,9 @@ def call
     private
 
     def delete_single_workload(workload)
-      return progress.puts("Workload '#{workload}' does not exist.") if cp.fetch_workload(workload).nil?
+      if cp.fetch_workload(workload).nil?
+        return progress.puts("Workload '#{workload}' does not exist in app '#{config.app}'.")
+      end
```

Comment:
All changes in this file are simply to add, e.g., " app 'my-app'", at the end of the progress messages, so that it's clear which app each item being deleted belongs to.